projects/m6/001-lotto-game/solutions/federicosiddi/lotto/cities.py

```diff
@@ -0,0 +1,30 @@
+class Cities:
+    """
+    Class used to store cities aka ruota names and includes methods to validate user city choice
+
+    """
+    CITIES = ["Bari", "Cagliari", "Firenze", "Genova", "Milano", "Napoli", "Palermo", "Roma", "Torino", "Venezia",
+              "Tutte"]
+
+    @staticmethod
+    def city_choice_info():
+        print("Type the corresponding city name to select the bill city")
+        print(*Cities.CITIES)
+
+    @staticmethod
+    def is_city_valid():
+        """
+        Function used to validate user city choice
+
+        :return: type str, returns a valid city name that is present in CITIES list variable
+        """
+        while True:
+            try:
+                city_choice = input("Enter value here -->: ")
+                while city_choice.capitalize() not in Cities.CITIES:
```

Comment:
Tipically in software development it is more common to verify strings by converting them both to lowercase